gobgp/cmd/mrt.go

```diff
@@ -174,7 +177,7 @@ func NewMrtCmd() *cobra.Command {
 					}
 				}
 			}
-			err := injectMrt(filename, count, skip, mrtOpts.Best)
+			err := injectMrt(filename, count, skip, &opts)
```

Comment:
I guess that the original code confused you. You don't need to pass anything here. injectMrt() can access to mrtOpts variant.

Reply:
I thought it would be more readable if all parameters (well, all related to cli input) that change function is acting on would be explictly passed to it.

I changed it in the first place because originally my code was just
  ```go  
    func injectMrt(filename string, count int, skip int, skipV4 bool, skipV6, onlyBest bool)
```
but then I thought if there will be some other filter added in the future then that would be pretty ugly way to pass extra option so I moved it into mrtOpts